nb/o.n.upgrader/src/org/netbeans/upgrade/Copy.java

```diff
@@ -119,67 +116,5 @@ private void copyFile(FileObject sourceFile) throws IOException {
         }
         FileUtil.copyAttributes(sourceFile, tg);        
     }
-    
-    public static void appendSelectedLines(File sourceFile, File targetFolder, String[] regexForSelection)
```

Comment:
this method is also not used. Here I removed the test too since this makes no deeper calls beside JDK code -> no real test coverage is lost